rust/Cargo.lock

```diff
@@ -28,9 +28,9 @@ dependencies = [
 
 [[package]]
```

Comment:
@cryptonemo this file should not be removed, we need to have a fixed Cargo.lock

Reply:
I'm not following.  Cargo.lock was updated to the latest deps, not removed.

Reply:
damn, I am not on my game today, there where so many deletions and got confused